### PR TITLE
P0 edit post&edit user

### PR DESCRIPTION
**Pull request by bhtibrewal**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -6,6 +6,7 @@
   --z-index-modal: 3;
   --user-dropdown-index: 2;
   --aside-z-index:1;
+  --z-index-dropdown: 1;
   /* --floating-btn-index:  */
```
Comment (pankaj-prog) [inline src/App.css:10]: remove commented code

File: src/assets/icons.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-    FaRegUser, FaRegComment, FaRegSmile, FaTelegramPlane, FaHeart, FaRegHeart, FaEye, FaEyeSlash, FaInfoCircle, FaExclamationCircle, FaCheckCircle, FaPlus
+    FaRegUser, FaRegComment, FaRegSmile, FaTelegramPlane, FaHeart, FaRegHeart, FaEye, FaEyeSlash, FaInfoCircle, FaExclamationCircle, FaCheckCircle, FaPlus, FaAngleRight, FaSearch, FaAngleDown, 
 } from "react-icons/fa";
-export { FiSettings, FiImage } from "react-icons/fi";
+export { FiSettings, FiImage, FiEdit, FiDelete } from "react-icons/fi";
 export { HiMenu } from 'react-icons/hi';
 export { MdGif } from 'react-icons/md';
-export { BsBookmark, BsBookmarkFill } from 'react-icons/bs';
+export { BsBookmark, BsBookmarkFill, BsThreeDotsVertical } from 'react-icons/bs';
 export { IoCloseCircleOutline } from 'react-icons/io5';
```
Comment (pankaj-prog) [inline src/assets/icons.js:9]: Like this, the way you are exporting all react icons from a single file

File: src/components/card/PostCard.jsx [hunk 3]
```diff
@@ -7,14 +7,20 @@ import {
   FaTelegramPlane,
   BsBookmark,
   BsBookmarkFill,
+  BsThreeDotsVertical,
+  FiEdit,
+  FiDelete,
 } from "../../assets/icons";
 import { useAuth, usePosts, useToast } from "../../contexts";
```
Comment (pankaj-prog) [inline src/components/card/PostCard.jsx:14]: use absolute imports

File: src/components/header/components/Search.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { getUsers } from "../../../services";
+import { UserSection } from "../../user_section/UserSection";
+import { FaSearch } from "../../../assets/icons";
```
Comment (pankaj-prog) [inline src/components/header/components/Search.jsx:4]: use absolute imports here as well

**Review by pankaj-prog — APPROVED** [review]

**Review by sri-rishi — APPROVED** [review]
Code looks good, feel free to merge

**Pull request merged**